src/uwtools/config/tools.py

```diff
@@ -205,19 +206,29 @@ def _realize_config_values_needed(input_obj: Config) -> bool:
     Print a report characterizing input values as complete, empty, or template placeholders.
 
     :param input_obj: The config to update.
+    :return: True
     """
     complete, empty, template = input_obj.characterize_values(input_obj.data, parent="")
-    log.info("Keys that are complete:")
-    for var in complete:
-        log.info(var)
+    if complete:
+        log.info("Keys that are complete:")
+        for var in complete:
+            log.info(var)
+    else:
+        log.info("No keys are complete.")
```

Comment:
Here and in the next two diff blocks, rather than having output like
```
[2024-01-10T21:29:20]     INFO Keys with unrendered Jinja2 variables/expressions:
[2024-01-10T21:29:20]     INFO
[2024-01-10T21:29:20]     INFO Keys that are set to empty:
[2024-01-10T21:29:20]     INFO 
```
when there are no such keys, make a more definitive statement like
```
[2024-01-10T21:29:20]     INFO No keys have unrendered Jinja2 variables/expressions.
[2024-01-10T21:29:20]     INFO No keys are set to empty.
```

Reply:
Worth the additional lines and work to tighten it up